crates/client/src/client/schema/snapshots/fuel_core_client__client__schema__block__tests__block_by_height_query_gql_output.snap

```diff
@@ -35,8 +35,6 @@ query($height: U32) {
         signature
       }
     }
-    transactions {
-      id
-    }
+    transactionIds
```

Comment:
why are we making a separate field for these ids?

Reply:
In this case we don't resolve the `transactions` field. Resolving of the field fetch al transactions from the storage.

<img width="642" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/3d856799-25df-421b-93bf-b086360a0a30">